libraries/AP_RCProtocol/AP_RCProtocol.cpp

```diff
@@ -238,6 +238,28 @@ void AP_RCProtocol::process_handshake( uint32_t baudrate)
   check for bytes from an additional uart. This is used to support RC
   protocols from SERIALn_PROTOCOL
  */
+void AP_RCProtocol::SerialConfig::apply_to_uart(AP_HAL::UARTDriver *uart) const
+{
+    uart->configure_parity(parity);
+    uart->set_stop_bits(stop_bits);
+    uart->set_options(uart->get_options() & ~masked_out_options);
```

Comment:
calling set_options twice reconfigures the hw twice
a bool for RXINV would avoid this

Reply:
Done